.github/workflows/reusable_checks.yml

```diff
@@ -277,7 +277,7 @@ jobs:
 
       - name: Taplo check
         run: |
-          taplo fmt --check
+          taplo fmt --check --diff
```

Comment:
out of the scope of this PR, but we should call `just` functions in the .yml files instead of duplicating code imho

Reply:
Agreed